wordpress-mobile/WordPress-Android
# Issue/5779 aztec media attributes

### Fix
When inserting an image from the WordPress media library, the image's HTML contains all custom attributes as described in https://github.com/wordpress-mobile/WordPress-Android/issues/5779.  The attributes included are `title`, `width`, and `height`.  The `alt text` attribute is not included since it doesn't exist in the [`MediaFile`](https://github.com/wordpress-mobile/WordPress-Utils-Android/blob/develop/WordPressUtils/src/main/java/org/wordpress/android/util/helpers/MediaFile.java) object even though that value can be set for the media file on the web.  The `alt text` attribute shown on the ***Image Settings*** screen is taken from the HTML when the [media is tapped](https://github.com/wordpress-mobile/WordPress-Android/blob/develop/libs/editor/WordPressEditor/src/main/java/org/wordpress/android/editor/AztecEditorFragment.java#L870) and not from the media file stored on the device.

### Test
0. Add custom attributes to image.
1. Go to ***Sites*** tab.
2. Tap ***Create*** floating button.
3. Tap content text field.
4. Tap ***Media*** format button.
5. Tap ***WordPress*** media picker button.
6. Tap image from library.
7. Tap ***HTML*** format button.
8. Notice `img` tag with custom attributes.
